### DISCUSSION TITLE
Issue with Receiving GitHub Copilot Subscription After Student Pack Approval

### DISCUSSION THREAD

**Discussion by tanmay-m-chaudhari** [community/community#134526] [GitHub Education]
### Hi everyone,

I successfully enrolled for the GitHub Student Developer Pack, and my request was **approved on July 27, 2024**. However, as of today, August 3, 2024, I have not received any email regarding my GitHub Copilot subscription.

Could you please assist me with this issue and let me know the status of my GitHub Copilot subscription? Any help or guidance you can provide would be greatly appreciated.

Thank you for your attention to this matter.

**Comment by davevad93** [comment]
Hi @TBHITSmeeey1911 ,

What's the state of your application? If it says "``Congrats! Your request for @TBHITSmeeey1911 was approved and your benefits will be available soon.``" it means you've been approved you still have to wait in order to use your benefits. 

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @TBHITSmeeey1911 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by tanmay-m-chaudhari** [comment]
nbm
